deployments/azuredeploylogs.json

```diff
@@ -0,0 +1,307 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters":
+    {
+        "ContainerName":
+        {
+            "defaultValue": "logs",
```

Comment:
Let's change it to a name that is more clear on the purpose of the container - if it's for backup logs, then something like "backup logs"